README.md

```diff
@@ -45,14 +47,38 @@ skipper make test
 make lint
 ```
 
+### Debug
+
+#### Bootstrap step
+
+Add `--debug-bootstrap` flag to the build command for avoiding machine reboot on bootstrap step completion. Useful for taking a snapshot of the appliance disk image before testing changes in the install ignition. 
```

Comment:
```suggestion
Add `--debug-bootstrap` flag to the build command to avoid machine reboot on bootstrap step completion. Useful for taking a snapshot of the appliance disk image before testing changes in the install ignition. 
```

Reply:
done